.github/security-insights.yml

```diff
@@ -0,0 +1,25 @@
+header:
+  schema-version: 2.0.0
+  last-updated: '2025-04-02'
+  last-reviewed: '2025-04-02'
+  project-si-source: https://github.com/open-telemetry/.github/security-insights.yml
+
+repository:
+  url: https://github.com/open-telemetry/opentelemetry-go
+  status: active
+  accepts-change-request: true
+  accepts-automated-change-request: true
+  core-team:
+    - name: Damien Mathieu
+      affiliation: Elastic
```

Comment:
Will these get auto-updated, or should we update our doc for adding new maintainers?

Reply:
I wonder if we could refer to the CODEOWNERS file.

https://github.com/open-telemetry/opentelemetry-go/blob/a5abd88e1d469a151fd5f8e5e49e0a060b29d03c/CODEOWNERS#L17